lib/hanami/cli/parser.rb

```diff
@@ -19,7 +19,7 @@ def self.call(command, arguments, names)
         OptionParser.new do |opts|
           command.options.each do |option|
             opts.on(*option.parser_options) do |value|
-              parsed_options[option.name.to_sym] = value
+              parsed_options[option.name.to_sym] = option.array? ? value.split(',') : value
```

Comment:
IMO we should rely in OptionParser features, this class accept Arrays, please check this code example:
https://ruby-doc.org/stdlib-2.4.2/libdoc/optparse/rdoc/OptionParser.html#class-OptionParser-label-Complete+example

It should be implemented here: https://github.com/hanami/cli/pull/14/files#diff-febef6f57837621c53d1777245717043R91

Reply:
good idea, I like it 👍